Packages/TARAPOTO-LAGUNA-AZUL-JUNGLE-PACKAGE/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Jungle">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="../../css/main.css">
+	<link rel="stylesheet" type="text/css" href="../../css/global.css">
+	<!-- PACKAGE STYLES -->
+	<link rel="stylesheet" type="text/css" href="../../css/packages.css">
+	<title>TARAPOTO LAGUNA AZUL JUNGLE PACKAGE</title>
+	<!-- Script for generating the data -->
+	<script src="../../js/Packages.js"></script>
+</head>
+<body>
+	<div class="first Cusco">
+		<div class=logo>
+			<img src="https://res.cloudinary.com/dlia7phdx/image/upload/v1564095577/ATG/atg-logo.png">
+		</div>
+		<div class="gradient">
+			<h3>TARAPOTO LAGUNA AZUL JUNGLE PACKAGE</h3>
+		</div>
+	</div>
+	<div class="packages">
+
+	</div>
+</body>
+</html>
```

Comment:
newline

Reply:
ill change all the html to have newline at the end of file